src/qt/forms/sendcoinsdialog.ui

```diff
@@ -617,7 +617,7 @@
         <x>0</x>
         <y>0</y>
         <width>830</width>
-        <height>68</height>
+        <height>104</height>
```

Comment:
What's the reason for 104px height here? Is it because we have more space now for the scroll area? Currently, this gives some whitespace below the recipient-section-devider (bordered line).